src/Controls/src/Core/Items/SelectableItemsView.cs

```diff
@@ -49,7 +49,17 @@ public SelectableItemsView()
 		public object SelectedItem
 		{
 			get => GetValue(SelectedItemProperty);
-			set => SetValue(SelectedItemProperty, value);
+			set
+			{
+				if (ReferenceEquals(SelectedItem, value))
```

Comment:
I wonder if this is really a bug?

If the user wants an item to be "re-selectable"
Then I think they just need to clear the selection on the CV 

If we want to support "reselection" then we should probably add a new api for reselecting an already selected item.

Reply:
Right this would also work

```
        async void OnCollectionViewSelectionChanged(object sender, SelectionChangedEventArgs e)
        {
            if (cv.SelectedItem == null)
                return;
            
            cv.SelectedItem = null;
        }
```

```
    <CollectionView Margin="20"
                    ItemsSource="{x:Static data:DogData.Dogs}"
                    ItemTemplate="{StaticResource AnimalTemplate}"
                    SelectionMode="Single"
                    x:Name="cv"
                    SelectionChanged="OnCollectionViewSelectionChanged" />   
```

But I think that many devs don't know that they can achieve it like this. So maybe a short note in the documentation would be a good idea?